From a02ae68534859e5e670969ee4b4cd1f42bb176c7 Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Mon, 16 Aug 2021 14:33:18 +0800
Subject: [PATCH 12/12] config-tools: add SECURITY_VM_FIXUP config

Add SECURITY_VM_FIXUP config for Security VM whether it needs to do
fixup for TPM2 and SMBIOS

Tracked-On: #6320

Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/hybrid.xml            | 1 +
 misc/config_tools/data/adl-s-crb/hybrid_rt.xml         | 1 +
 misc/config_tools/data/adl-s-crb/industry.xml          | 1 +
 misc/config_tools/data/adl-s-crb/logical_partition.xml | 1 +
 misc/config_tools/data/icx-rvp/industry.xml            | 1 +
 5 files changed, 5 insertions(+)

diff --git a/misc/config_tools/data/adl-s-crb/hybrid.xml b/misc/config_tools/data/adl-s-crb/hybrid.xml
index 2ab65b994..ae5e5aeb9 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid.xml
@@ -16,6 +16,7 @@
             <MULTIBOOT2>y</MULTIBOOT2>
             <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
             <ENFORCE_TURNOFF_GP>y</ENFORCE_TURNOFF_GP>
+            <SECURITY_VM_FIXUP>n</SECURITY_VM_FIXUP>
             <RDT>
                 <RDT_ENABLED>n</RDT_ENABLED>
                 <CDP_ENABLED>n</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
index f66cfa95c..3f94d6581 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid_rt.xml
@@ -15,6 +15,7 @@
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
       <ENFORCE_TURNOFF_GP>y</ENFORCE_TURNOFF_GP>
+      <SECURITY_VM_FIXUP>n</SECURITY_VM_FIXUP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>y</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/industry.xml b/misc/config_tools/data/adl-s-crb/industry.xml
index d4266966e..4a47d3bd2 100644
--- a/misc/config_tools/data/adl-s-crb/industry.xml
+++ b/misc/config_tools/data/adl-s-crb/industry.xml
@@ -15,6 +15,7 @@
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
       <ENFORCE_TURNOFF_GP>y</ENFORCE_TURNOFF_GP>
+      <SECURITY_VM_FIXUP>n</SECURITY_VM_FIXUP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>y</CDP_ENABLED>
diff --git a/misc/config_tools/data/adl-s-crb/logical_partition.xml b/misc/config_tools/data/adl-s-crb/logical_partition.xml
index 526dd815c..2f75d1e09 100644
--- a/misc/config_tools/data/adl-s-crb/logical_partition.xml
+++ b/misc/config_tools/data/adl-s-crb/logical_partition.xml
@@ -15,6 +15,7 @@
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
       <ENFORCE_TURNOFF_GP>y</ENFORCE_TURNOFF_GP>
+      <SECURITY_VM_FIXUP>n</SECURITY_VM_FIXUP>
       <RDT>
         <RDT_ENABLED>n</RDT_ENABLED>
         <CDP_ENABLED>n</CDP_ENABLED>
diff --git a/misc/config_tools/data/icx-rvp/industry.xml b/misc/config_tools/data/icx-rvp/industry.xml
index ba921b9fc..7b6eebc4e 100644
--- a/misc/config_tools/data/icx-rvp/industry.xml
+++ b/misc/config_tools/data/icx-rvp/industry.xml
@@ -15,6 +15,7 @@
       <MULTIBOOT2>y</MULTIBOOT2>
       <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
       <ENFORCE_TURNOFF_GP>n</ENFORCE_TURNOFF_GP>
+      <SECURITY_VM_FIXUP>n</SECURITY_VM_FIXUP>
       <RDT>
 	<RDT_ENABLED>n</RDT_ENABLED>
 	<CDP_ENABLED>y</CDP_ENABLED>
-- 
2.25.1

